Workaround: IT on prod timeout waiting for queues (#4299) (#4318)
DataBiosphere/azul: OPERATOR.rst

Once the base branch is restored, the ``Reopen PR`` button should again be
clickable on the chained PR.

Integration test time out
^^^^^^^^^^^^^^^^^^^^^^^^^

> This can happen when IT randomly selects bundles that have a large amount of

Reviewer: ```suggestion
This can happen in the rare occasion that the IT's random selection of bundles happens to pick predominantly large bundles
```

> This can happen in the rare occasion that the IT's random selection of bundles

Reviewer: ```suggestion
This can happen on the rare occasion that the IT's random selection of bundles
```

> entities that need to be divided before they can be indexed. In the AWS console,

Reviewer: Entities aren't being divided. Bundles are.

> happens to pick predominantly large bundles that need to be divided before they

Reviewer: ```suggestion
happens to pick predominantly large bundles that need to be partitioned before they
```

> can be indexed. In the AWS console, run the CloudWatch query below with the

Reviewer: ```suggestion
can be indexed. In the AWS console, run the CloudWatch Insights query below with the
```

> indexer log groups selected to see how many divisions have occurred::

Reviewer: Add note that not only bundles can be divided into partitions, but also partitions into sub-partitions. Technically, bundles are partitions with an empty prefix.

    fields @timestamp, @log, @message
    | filter @message like 'Dividing partition'
    | parse 'Dividing partition * of bundle *, version *, with * entities into * sub-partitions.' as partition, bundle, version, enities, subpartitions
    | display partition, bundle, version, enities, subpartitions
    | stats count(@requestId) as total_count by bundle, partition
    | sort total_count desc
    | sort @timestamp desc
    | limit 1000

> Note that when partitions are being divided, errors of exceeded rate & quota

Reviewer: Same linguistic inconsistency as above. A whole is divided into parts.

> Note that when bundles are being divided, errors of exceeded rate & quota

Reviewer: ```suggestion
Note that when bundles are being partitioned, errors of exceeded rate & quota
```

Once the base branch is restored, the ``Reopen PR`` button should again be
clickable on the chained PR.

Integration test time out
^^^^^^^^^^^^^^^^^^^^^^^^^

A warning sign that IT will likely time out is when Azul starts dividing up

> bundle partitions. Run the BigQuery query below against the indexer log groups

Reviewer 2: ```suggestion
bundle partitions. In the CloudWatch console run the query below with the indexer log groups selected
```

to see if this is the case::

    fields @timestamp, @message
    | filter @message like 'Dividing partition'
    | sort @timestamp desc
    | limit 20

> 

Reviewer 2: It feels to me like this is a redundant step, as it is now it implicitly implies any `Dividing partition` messages are a warning sign, is that the case?
